query-node/schemas/forum.graphql

```diff
@@ -113,8 +113,12 @@ type ForumThread @entity {
 
   "Assigned thread tags"
   tags: [ForumThreadTag!]
+
+  "New tag/label properties"
+  newTags: [Tag!]
```

Comment:
We will need to merge those functionalities actually, but I'm fine with this as a temporary measure to not bloat this PR too much. Once we're done with this PR we should most likely remove all the old tags related stuff